app/assets/javascripts/discourse/tests/acceptance/loading-indicator-test.js

```diff
@@ -96,4 +97,22 @@ acceptance("Page Loading Indicator", function (needs) {
     assert.strictEqual(currentRouteName(), "about");
     assert.dom("#main-outlet section.about").exists();
   });
+
+  test("it only performs one slide during nested loading events", async function (assert) {
+    this.siteSettings.page_loading_indicator = "slider";
+
+    await visit("/");
+
+    const events = [];
+    const service = getOwner(this).lookup("service:loading-slider");
+    service.on("stateChanged", (loading) => {
+      events.push(loading);
+    });
+
+    visit("/u/eviltrout/activity");
+
+    await settled();
```

Comment:
Any reason it's split into those two lines (instead of `await visit()`?)

Reply:
Nope!

```suggestion
    await visit("/u/eviltrout/activity");
```